docker-compose-using-tube-cheese/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+services:
```

Comment:
I think that can be good to add a custom `bridge` network and avoid to use the default `bridge`.